core/app/models/spree/refund.rb

```diff
@@ -29,26 +26,32 @@ def total_amount_reimbursed_for(reimbursement)
       end
     end
 
+    def perform_and_save!
+      perform!
+      save!
```

Comment:
`perform!` calls `update!`, so we don't need the `save!` here, right?

With the current state of this PR we could possibly even get rid of `perform_and_save!` and just have `perform!`.  But for now (until we add a `state` column) I personally think it's worth having `perform_and_save!`.